runtime/include/listener_thread.h

```diff
@@ -5,7 +5,7 @@
 #include "generic_thread.h"
 #include "module.h"
 
-#define LISTENER_THREAD_CORE_ID 0
+#define LISTENER_THREAD_CORE_ID 1
```

Comment:
What was the reason to make this change?

Reply:
Oh I think I got it, to leave core 0 for the system etc.

Reply:
Yep, that's pretty much it!